legacy/i18n/locales/en-US.json

```
@@ -3,7 +3,8 @@
   "menu.allWallets": "All Wallets",
   "menu.catalystVoting": "Catalyst Voting",
   "menu.settings": "Settings",
-  "menu.faq": "FAQ",
+  "menu.faqTitle": "Any Questions?",
+  "menu.faqLink": "ASK OUR SUPPORT TEAM",
```

I'd suggest capitalizing on the view only.
```suggestion
  "menu.faqLink": "Ask our support team",
```